xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,428 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+  type="text/xml"
+  title="Profiling step"?>
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<!--
+FATE#320599: Pacemaker Remote
+-->
+<?provo dirname="nfs_quick/"?>
+<article version="5.0" xml:lang="en" xml:id="art_sle_ha_pmremote"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:dm="urn:x-suse:ns:docmanager"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>&paceremote;</title>
+  <subtitle>&productname; &productnumber;</subtitle>
+  <info>
+   <productnumber>&productnumber;</productnumber>
+   <productname>&productname;</productname>
+   <date><?dbtimestamp format="m/d/Y X"?></date>
+   <abstract>
+    <para>
+     This document guides you through the setup of a cluster with virtual
+     guest nodes, managed by &pace; and &pmremote;.
```

Comment:
Actually it's pretty common to run pacemaker_remote on baremetal:
https://github.com/ClusterLabs/pacemaker/blob/master/doc/Pacemaker_Remote/en-US/Ch-Baremetal-Tutorial.txt